lib/fathead/scala/parse.py

```diff
@@ -0,0 +1,75 @@
+from bs4 import BeautifulSoup
+import os
+
+BASE_DIRECTORY = "./download/api/scala"
+BASE_URL = "https://www.scala-lang.org/api/current/"
+
+
+def get_all_files():
+	doc_files = []
+
+	for (path, dirs, files) in os.walk(BASE_DIRECTORY):
```

Comment:
@sam09 any reason we're not grabbing all the files in all the subdirectories?

Seems like we're only scratching the surface of the Scala API!